shared/desktop/app/menu-bar.desktop.tsx

```diff
@@ -10,10 +10,16 @@ import {resolveRoot, resolveImage, resolveRootAsURL} from './resolve-root.deskto
 import {showDevTools, skipSecondaryDevtools} from '../../local-debug.desktop'
 import getIcons from '../../menubar/icons'
 import {workingIsDarkMode} from '../../util/safe-electron.desktop'
+import os from 'os'
 
 const htmlFile = resolveRootAsURL('dist', `menubar${__DEV__ ? '.dev' : ''}.html?param=menubar`)
 
+// support dynamic dark mode system bar in big sur
+const useImageTemplate = os.platform() === 'darwin' && parseInt(os.release().split('.')[0], 10) >= 20
```

Comment:
do we need to check `null` from `os.release()`?

Reply:
no, its typed as a string and the docs agree

Reply:
(also we do this elsewhere)